hail/python/hail/methods/statgen.py

```diff
@@ -364,10 +368,35 @@ def linear_regression_rows(y, x, covariates, block_size=16) -> hail.Table:
 
```

Comment:
It appears you dropped the check of whether `y` is empty in the case that it is a non-empty list: e.g. `[[]]` and `[[a],[]]` should both raise errors.